quickstart.sh

```diff
@@ -7,27 +7,33 @@ set -euo pipefail
 # to tell the user to use files that we've cleaned up.
 DO_CLEANUP=0
 
+color_title=$(tput setaf 7; tput bold)
+color_dim=$(tput setaf 8)
+color_good=$(tput setaf 2)
+color_bad=$(tput setaf 1)
+color_warn=$(tput setaf 3)
+color_reset=$(tput sgr0)
+
 usage() {
     cat <<EOF
-$0
+${color_title}$0${color_reset}
 Downloads the latest version of the Zipkin Server executable jar
 
-$0 GROUP:ARTIFACT:VERSION:CLASSIFIER TARGET
+${color_title}$0 GROUP:ARTIFACT:VERSION:CLASSIFIER TARGET${color_reset}
 Downloads the "VERSION" version of GROUP:ARTIFACT with classifier "CLASSIFIER"
 to path "TARGET" on the local file system. "VERSION" can take the special value
 "LATEST", in which case the latest Zipkin release will be used. For example:
 
-$0 io.zipkin.java:zipkin-autoconfigure-collector-kafka10:LATEST:module kafka10.jar
```

Comment:
The `kafka10` collector seems to be dead (?), so updating this seems to make sense.